.github/workflows/publish.yaml

```diff
@@ -147,26 +152,104 @@ jobs:
             --build_tag_filters="-nokokoro,-nopresubmit,-requires-gpu-nvidia" \
             --google_credentials="${{ steps.auth.outputs.credentials_file_path }}" \
             --remote_cache="https://storage.googleapis.com/${{ vars.BAZEL_CACHE_BUCKET }}/${{ github.job }}" \
-            --config="linux_${{ matrix.arch }}" \
+            --config="${{ matrix.sys }}_${{ matrix.arch }}" \
             -- \
             --output_dir="${{ github.workspace }}/dist/"
 
     - name: Upload Python package
       uses: actions/upload-artifact@v4.3.3
       with:
-        name: python-package-distributions-${{ matrix.arch }}
+        name: python-package-distributions-${{ matrix.sys }}-${{ matrix.arch }}
+        path: dist/*.whl
+
```

Comment:
Why manylinux is still there? It didn't work, let's discard it for now completely